test/js-scripts/src/sign-wrapped-typed-data.ts

```diff
@@ -74,40 +71,31 @@ async function signWrappedTypedData(): Promise<void> {
         const appDomain = {
             name: appDomainName,
             version: appDomainVersion,
+            verifyingContract: appVerifyingContract,
             chainId: 31337, // Default Anvil chain ID
+            // salt: '0x0000000000000000000000000000000000000000000000000000000000000000' as `0x${string}`,
         }
         const verifierDomain = {
             name: VERIFIER_DOMAIN_NAME,
             version: VERIFIER_DOMAIN_VERSION,
             verifyingContract: verifyingContract,
             chainId: 31337, // Default Anvil chain ID
-            salt: '0x' as `0x${string}`,
+            salt: '0x0000000000000000000000000000000000000000000000000000000000000000' as `0x${string}`,
```

Comment:
so just doing '0x'was causing a diff hash?

Reply:
no, this one was fine, it was in the app domain above, in solidity we didn't have the salt at all